### DISCUSSION TITLE
Are you planning to support Nuxt?

### DISCUSSION THREAD

**Discussion by RadoSabo** [highlight/highlight#6095] [Q&A]
I saw there is support for Vue and for Next, so I wanted to know if there is any plan to add Nuxt? 
Thank you.

**Comment by jay-khatri** [comment]
Hi there. We're working on [Remix](https://github.com/highlight/highlight/pull/5992) at the moment, but we can definitely consider Nuxt next. I'll leave this discussion open to gauge interest.

**Comment by shintaii** [comment]
To be fair, having Vue support is already 99% right.. I'm using Nuxt2 and installed it as a plugin, and it works right away.

**Reply by notflip** [reply]
@shintaii Did you have to do anything special inside the plugin? I'm using the following, but inside my highlight dashboard I still see install pending

```
import {H} from 'highlight.run';

export default defineNuxtPlugin(() => {
    const config = useRuntimeConfig()

    H.init('123456', {
        environment: config.public.env as string ?? 'dev',
        enableCanvasRecording: false,
        enablePerformanceRecording: false,
        networkRecording: {
            enabled: true
        }
    })
    return {
        provide: {
            H,
        },
    }
})

```

**Comment by RadoSabo** [comment]
> To be fair, having Vue support is already 99% right.. I'm using Nuxt2 and installed it as a plugin, and it works right away.

but is the backend logging working too? Vue will cover only FE logs, or am I wrong?
